client/camera.lua

```diff
@@ -340,24 +326,21 @@ end)
 
 RegisterNetEvent("Mic:ToggleBMic", function()
     if not holdingBmic then
-        RequestModel(GetHashKey(bmicModel))
-        while not HasModelLoaded(GetHashKey(bmicModel)) do
-            Wait(100)
-        end
-        local plyCoords = GetOffsetFromEntityInWorldCoords(GetPlayerPed(PlayerId()), 0.0, 0.0, -5.0)
-        local bmicspawned = CreateObject(GetHashKey(bmicModel), plyCoords.x, plyCoords.y, plyCoords.z, true, true, false)
+        lib.requestModel(bmicModel)
+        local plyCoords = GetOffsetFromEntityInWorldCoords(GetPlayerPed(cache.playerId), 0.0, 0.0, -5.0)
+        local bmicspawned = CreateObject(bmicModel, plyCoords.x, plyCoords.y, plyCoords.z, true, true, false)
```

Comment:
Use joaat on bmicModel for this to work

Reply:
doesnt this already take place with lib.requestModel?